test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -124,24 +128,25 @@ class thread_context {
     /*
      * Generic update function, takes a collection_id and key, will generate the value.
      *
-     * Returns true if it successfully updates the key, false if it receives rollback from the API.
+     * Returns a boolean indicating if a rollback is required.
      */
     bool update(scoped_cursor &cursor, uint64_t collection_id, const std::string &key);
 
     /*
      * Generic insert function, takes a collection_id and key_id, will generate the value.
      *
-     * Returns true if it successfully inserts the key, false if it receives rollback from the API.
+     * Returns a boolean indicating if a rollback is required.
```

Comment:
We need to say the value of the boolean to avoid confusion.
`Returns true if...`